utils/build.config

```diff
@@ -5,5 +5,5 @@ component=cart
 OFI = v1.7.0rc3
 OMPI = 373098d8ae0053af85cc1d49a58dbe933a31a50a
 OPENPA = 8e1e74feb22d2e733f34a96e6c7834fed3073c52
-MERCURY = v1.0.1
+MERCURY = 9f9dd80164a2b14b184f2b373efeb50a5fc80dc5
```

Comment:
This is a pity.  Has the problem that requires this been reported upstream?

Reply:
It's been landed upstream which is why we're updating.

Reply:
Indeed, I did notice that also.